src/hooks/useMassSelection.ts

```diff
@@ -66,6 +68,16 @@ export const useMassSelection = (
     }
   };
 
+  const selectMultipleIds = (newIds: string[]) => {
```

Comment:
It's probably good to add tests for this method. And `deselectMultipleIds` if you're not able to remove it and just use `deselectAll`.